src/components/chip/chip.e2e.ts

```diff
@@ -0,0 +1,45 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('limel-chip', () => {
+    let page;
+
+    beforeEach(async () => {
+        page = await newE2EPage();
+    });
+
+    it('should render with the correct text', async () => {
+        await page.setContent('<limel-chip text="Test chip"></limel-chip>');
+        await page.waitForChanges();
+
+        const element = await page.find('limel-chip >>> .chip');
+        expect(element.textContent).toEqual('Test chip');
+    });
+
+    it('should render as a link when the link prop is provided', async () => {
+        await page.setContent(
+            '<limel-chip text="Test chip" link="{ href: \'https://example.com\' }"></limel-chip>',
+        );
+        await page.waitForChanges();
+
+        const element = await page.find('limel-chip >>> a.chip');
+        expect(element.getAttribute('href')).toEqual('https://example.com');
+    });
```

Comment:
<img width="529" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/21972a2c-1408-4702-8605-734ec1da0b15">
Don't know how to fix this

Reply:
The fact that the actual response has `tabindex="0"` should be easy to fix by just adding that to the expected output. (It looks like this comment is placed on the wrong test? If it's not, then I have no idea, to be honest.)

The fact that the actual response is _missing` the `href="https://example.com"` on the other hand, indicates either that the functionality we are testing for doesn't work (unlikely, since it seems to work in the PR docs), or that we need to wait for it to re-render or something.

I'm sure I or someone else can have a look 👍